packages/code-studio/src/styleguide/CheckboxGroups.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Checkbox, CheckboxGroup, Flex, Text } from '@deephaven/components';
+import SampleSection from './SampleSection';
+
+export function CheckboxGroups(): JSX.Element {
+  return (
+    <SampleSection name="checkbox-groups">
+      <h2 className="ui-title">Checkbox Groups</h2>
+      <Flex gap="size-100" gridColumn="span 3" height="100%">
+        <Flex direction="column">
+          <Text>Single Child</Text>
+          <CheckboxGroup aria-label="Single Child">
+            <Checkbox checked={false}>Aaa</Checkbox>
+          </CheckboxGroup>
+        </Flex>
+
+        <Flex direction="column">
+          <Text>Multiple Children</Text>
+          <CheckboxGroup aria-label="Multiple Children">
+            <Checkbox checked={false}>Aaa</Checkbox>
```

Comment:
If you use the `SpectrumCheckbox`, you shouldn't need to set `checked` for uncontrolled component.